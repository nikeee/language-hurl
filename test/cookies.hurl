# Run a GET request with cookies section:
GET https://example.org/index.html
[Cookies]
theme: light
sessionToken: abc123

# Run the same GET request with a header:
GET https://example.org/index.html
Cookie: theme=light; sessionToken=abc123
