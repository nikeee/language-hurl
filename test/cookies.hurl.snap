># Run a GET request with cookies section:
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hurl comment.line.number-sign.hurl
>GET https://example.org/index.html
#^^^ source.hurl keyword.control.method.hurl
#   ^ source.hurl
#    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hurl string.unquoted.url.hurl string.unquoted.url.hurl
>[Cookies]
#^^^^^^^^^ source.hurl entity.name.section.hurl
>theme: light
#^^^^^ source.hurl entity.name.tag.header.hurl entity.name.tag.header.hurl
#     ^ source.hurl entity.name.tag.header.hurl punctuation.separator.key-value.hurl
#      ^ source.hurl entity.name.tag.header.hurl
#       ^^^^^ source.hurl entity.name.tag.header.hurl
>sessionToken: abc123
#^^^^^^^^^^^^ source.hurl entity.name.tag.header.hurl entity.name.tag.header.hurl
#            ^ source.hurl entity.name.tag.header.hurl punctuation.separator.key-value.hurl
#             ^ source.hurl entity.name.tag.header.hurl
#              ^^^^^^ source.hurl entity.name.tag.header.hurl
>
># Run the same GET request with a header:
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hurl comment.line.number-sign.hurl
>GET https://example.org/index.html
#^^^ source.hurl keyword.control.method.hurl
#   ^ source.hurl
#    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hurl string.unquoted.url.hurl string.unquoted.url.hurl
>Cookie: theme=light; sessionToken=abc123
#^^^^^^ source.hurl entity.name.tag.header.hurl entity.name.tag.header.hurl
#      ^ source.hurl entity.name.tag.header.hurl punctuation.separator.key-value.hurl
#       ^ source.hurl entity.name.tag.header.hurl
#        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hurl entity.name.tag.header.hurl
>